packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -58,10 +59,12 @@ const ConnectorDetails = () => {
       .json<ConnectorDTO>();
     toast.success(t('connector_details.connector_deleted'));
 
+    await mutateGlobal('/api/connectors');
```

Comment:
is there a better way like auto mutate after/with api request? is too easy to forget to call this function

Reply:
it'll be great if we can use only one `mutate` (i know they are different data source now. can try to refactor later)

Reply:
discussed offline, solve later